src/burn/engine/plan.cpp

```diff
@@ -1920,6 +1927,10 @@ static HRESULT GetActionDefaultRequestState(
 
     switch (action)
     {
+    case BOOTSTRAPPER_ACTION_CACHE:
+        *pRequestState = BOOTSTRAPPER_REQUEST_STATE_CACHE;
```

Comment:
This needs to be like MODIFY, it depends on the current state of the package.  If the package is PRESENT and the default action is CACHE, then the package would get uninstalled.